great_expectations/checkpoint/util.py

```diff
@@ -443,6 +442,17 @@ def get_updated_action_list(
     return list(base_action_list_dict.values())
 
 
+def batch_request_in_validations(
+    validations: Optional[List[dict]] = None,
+) -> bool:
+    if validations is not None:
+        for idx, val in enumerate(validations):
```

Comment:
Nit - Don't need to enumerate as we aren't looking at indices

Reply:
@cdkini I fixed it universally in this module -- thanks for catching.